### ISSUE TITLE
Data for very large projects cannot be downloaded

### ISSUE THREAD

**Issue by tinok** [kobotoolbox/kobocat#250]
* Regular data exports: Celery seems to kill the process before the file is created
* New formpack exports: Server responds with gateway timeout after 1 minute

See [this example](https://www.flowdock.com/app/kobotoolbox/kobo/threads/YlzcOblRQ9Cu4Hh4iT9oxCNCQm6)

**Comment by jnm** [comment]
Split the formpack timeout to a separate issue: #255 
This issue now tracks increasing the Celery timeout

**Comment by jnm** [comment]
Will need to apprise OCHA of https://github.com/UN-OCHA/kobo-dockers/pull/6 once this is merged.
